### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Jan-Bart**

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  } = config;
+
+  const {
+    ready = [],
+    alive = [],
+  } = hooks;
+
+  async function runAliveChecks(req, res) {
+    try {
+      await Promise
```
Comment (stefanpante) [inline src/controller.js:13]: why combine await with then they do the same thing?

Comment (stefanpante) [reply src/controller.js:13]: you could just write this as 
```javascript
try {
  await Promise.all(alive.map(check => check()));
  return res.json({...})
} catch (error) {
    console.error(`PROBES_ERROR_2: ${error.message || ''}`);
    return res.status(error.status || 500).json({ message: error.message || '' });
}
```

File: src/controller.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+  }
+
+  async function runReadyChecks(req, res) {
+    try {
+      await Promise
+        .all(
+          ready.map(check => check())
+        )
+        .then(() => {
```
Comment (stefanpante) [inline src/controller.js]: same remark as above

**Review by stefanpante — CHANGES_REQUESTED** [review]
rewrite your Promise handling. It contains redundant logic.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+language: node_js
+
+node_js:
+  - "8"
+  - "10"
+  - "11"
```
Comment (vademo) [inline .travis.yml:7]: 12, 13

**Review by vademo — APPROVED** [review]
Only minor remarks ;)

**Review by stefanpante — APPROVED** [review]

**Pull request merged**